### PR TITLE
Add 'ps' output to /processes

### PR DESCRIPTION
**Pull request by mia-pi-git**
Zarel mentioned that we should do this a while ago (iirc? thought it was brought up), and it's a useful enough tool for debugging it couldn't hurt to have it here.

### CODE REVIEW

File: server/chat-commands/admin.ts [hunk 1]
```diff
@@ -602,16 +608,57 @@ export const commands: ChatCommands = {
 			this.checkCan('lockdown');
 		}
 
-		let buf = `<strong>${process.pid}</strong> - Main<br />`;
+		const processes = new Map<string, ProcessData>();
+
+		const psOutput = child_process.execSync('ps -o pid,%mem,time,command', {cwd: `${__dirname}/../..`}).toString();
```
Comment (Zarel) [inline server/chat-commands/admin.ts:613]: Is `cwd` necessary? `ps` isn't touching the filesystem, is it?

Comment (mia-pi-git) [reply server/chat-commands/admin.ts:613]: I'm not quite sure? I figured better safe than sorry, especially since `bash` does it above as well.

Comment (Zarel) [reply server/chat-commands/admin.ts:613]: Also, why not `%cpu`? That's the whole reason we wanted it in he first place, right?

Comment (mia-pi-git) [reply server/chat-commands/admin.ts]: lol yeah i switched them - fixed.

File: server/chat-commands/admin.ts [hunk 2]
```diff
@@ -602,16 +608,57 @@ export const commands: ChatCommands = {
 			this.checkCan('lockdown');
 		}
 
-		let buf = `<strong>${process.pid}</strong> - Main<br />`;
+		const processes = new Map<string, ProcessData>();
+
+		const psOutput = child_process.execSync('ps -o pid,%cpu,time,command', {cwd: `${__dirname}/../..`}).toString();
+		const rows = psOutput.split('\n').slice(1); // first line is the title
```
Comment (Zarel) [inline server/chat-commands/admin.ts:614]:
```suggestion
		const rows = psOutput.split('\n').slice(1); // first line is the table header
```

File: server/chat-commands/admin.ts [hunk 3]
```diff
@@ -602,16 +608,57 @@ export const commands: ChatCommands = {
...
+		buf += `<strong>Other processes:</strong><br />`;
+
+		for (const [pid, process] of processes) {
+			buf += `<strong>${pid}</strong> - <code>${process.cmd}</code>`;
+			if (process.cpu) buf += ` (CPU: ${process.cpu}`;
+			if (process.time) {
+				buf += `${process.cpu ? `, ` : ' ('}time: ${process.time})`;
+			}
+			buf += `<br />`;
+		}
```
Comment (Zarel) [inline server/chat-commands/admin.ts:661]: This should be in a readmore; it's too spammy to show by default.

Comment (mia-pi-git) [reply server/chat-commands/admin.ts]: Done.

Comment (Zarel) [reply server/chat-commands/admin.ts]: It plausibly shouldn't be shown at all, but I guess we can try it out for now and see how it goes.

**Comment by Zarel** [comment]
This is exactly what I wanted! Thanks a ton!

**Pull request merged**